tasks/variables.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Include OS-specific variables (Debian).
+  include_vars: "{{ ansible_distribution }}-{{ ansible_distribution_version.split('.')[0] }}.yml"
+  when: ansible_os_family == 'Debian'
+
+- name: Define certbot package version.
+  set_fact:
+    certbot_version: "{{ __certbot_version }}"
```

Comment:
why the `__` prefix? is this some sort of convention?

Reply:
Well, it is not a convention but I saw it in the Geerling Guy roles. He uses the `__` to show that this var is *private* have a *default* value that *depends on the environment*. The var `certbot_version` is the one used in the tasks.

Reply:
then wouldn't it be enough to prepende the name with just a single underscore? that's more common, right?

Reply:
Right, now we use only one underscore to the SO default vars.